src/components/HTMLEngineProvider/HTMLRenderers/AnchorRenderer.tsx

```diff
@@ -30,6 +30,7 @@ function AnchorRenderer({tnode, style, key}: AnchorRendererProps) {
     const internalNewExpensifyPath = Link.getInternalNewExpensifyPath(attrHref);
     const internalExpensifyPath = Link.getInternalExpensifyPath(attrHref);
     const isVideo = attrHref && Str.isVideo(attrHref);
+    const containsImageLink = tnode.tagName === 'a' && tnode.children.some((child) => child.tagName === 'img');
```

Comment:
Could you please update the variable name to make it clear? In this case, since tag A includes an image so it should be renamed to isLinkHasImage.
```suggestion
    const isLinkHasImage = tnode.tagName === 'a' && tnode.children.some((child) => child.tagName === 'img');
```

Reply:
it's done